src/schemas/space.json

```diff
@@ -20,6 +20,11 @@
           "title": "about",
           "maxLength": 160
         },
+        "terms": {
+          "type": "string",
+          "title": "terms",
+          "format": "uri"
```

Comment:
Can you add this format on "website" field too?

Reply:
`uri` will not be invalid without`*://` at the beginning. Any what these other ones mean?
- uri-reference: URI reference, including full and relative URIs.
- uri-template: URI template according to RFC6570
Source: https://github.com/ajv-validator/ajv-formats

Reply:
I don't think this will work because domain field isn't really a `uri`.

Reply:
I mean `website` field, it's another one

Reply:
Oh, I see